### ISSUE TITLE
Check file uploads

### ISSUE THREAD

**Issue by Compro-Prasad** [lugnitdgp/nitdgp_website#57]
Allow documents and compressed files only.

**Comment by realsdx** [comment]
You mean MIME Type checking ?

**Comment by Compro-Prasad** [comment]
Yes. But I am unsure if it reveals the extension's format or it actually reads file headers(first few bytes) like `file` command does. Because some people try to re-upload the file by just changing the extension which is not preferable from many standpoints.

**Comment by realsdx** [comment]
Then, probably there is no better way. Also, as files are uploaded by staffs, not normal user and uploaded files are **never executed** (dajngo even never executes any arbitrary python files). So extension checking is good enough. MIME checking will just allow some extra layer. But, either way sever is never effected.

**Comment by Compro-Prasad** [comment]
Then do whatever is best for now.
